### PR TITLE
Initial commit of migrating adoptium-packages-linux

### PR DESCRIPTION
**Pull request by sophia-guo**
Initial migration from https://github.com/aahlenst/adoptium-packages-linux
Add a jenkins file as a start testing point

Co-authored-by: Andreas Ahlenstorf <andreas@ahlenstorf.ch>
Co-authored-by: Sophia Guo <sophia.gwf@gmail.com>

### CODE REVIEW

**Comment by sophia-guo** [comment]
Test: https://ci.adoptopenjdk.net/view/work-in-progress/job/adoptium-packages-linux-pipeline/17

File: linuxNew/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            steps {
+                dir('linuxNew') {
+                    buildAndTest()
+	            }
+            }
+        }
+        // For now looks like upload is combined with build and tests
+        stage('Upload') {
+            steps {
+                upLoadAritifact()
```
Comment (smlambert) [inline linuxNew/Jenkinsfile:23]: minor name update: please change upLoadAritifact to uploadArtifacts

**Review by smlambert — COMMENTED** [review]
LGTM aside from minor spelling fix requested.  If you have not already done so, can you ensure that to use co-authored-by to reference Andreas's github account also (https://docs.github.com/en/github/committing-changes-to-your-project/creating-a-commit-with-multiple-authors).  Thanks @sophia-guo !

**Review by jerboaa — APPROVED** [review]
Seems fine to me.

File: linuxNew/Jenkinsfile (hunk 1 above)
Comment (sophia-guo) [reply linuxNew/Jenkinsfile]: Thanks! Updated.

**Review by smlambert — APPROVED** [review]
thanks @sophia-guo !

**Pull request merged**